Resources/Prototypes/Floof/Entities/Clothing/Uniforms/misc.yml

```
@@ -12,3 +12,136 @@
     sprite: Floof/Clothing/Uniforms/loincloth.rsi
     equipDelay: 0
     unequipDelay: 0
+
```

```suggestion
    unequipDelay: 0
  - type: Tag #DeltaV, needed for species with nonhuman legs/can only wear skirts
    tags:
    - Skirt

```
This is needed so harpies can wear loincloth.`

---

Ah! Should I actually add that to most of the uniforms that don't cover the legs? Cause the thongs are pretty small uniform sprites that don't really go lower then the pelvis.